### PR TITLE
feat(api-token): Add ability to use API tokens but not disable "api-token-auth"

### PR DESCRIPTION
**Pull request by kiblik**
`DD_API_TOKENS_ENABLED` is useful for disabling all API tokens.

From now, `DD_API_TOKEN_AUTH_ENDPOINT_ENABLED` might be used when Tokens are needed but only "user+pass -> token" endpoint needs to be disabled.

### CODE REVIEW

**Review by mtesauro — APPROVED** [review]
Approved

File: docker-compose.override.dev.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ services:
       DD_ADMIN_USER: "${DD_ADMIN_USER:-admin}"
       DD_ADMIN_PASSWORD: "${DD_ADMIN_PASSWORD:-admin}"
       DD_EMAIL_URL: "smtp://mailhog:1025"
+      DD_API_TOKEN_AUTH_ENDPOINT_ENABLED: "False"
```
Comment (cneill) [inline docker-compose.override.dev.yml:12]: Is there a specific reason to disable this in the `dev` override config and not others? Seems like a development environment would be pretty low-risk.

Comment (kiblik) [reply docker-compose.override.dev.yml:12]: I'm sorry. It is leftover from testing. I will remove it.

Comment (kiblik) [reply docker-compose.override.dev.yml:12]: I'm sorry. It is leftover from testing. I will remove it.

**Review by grendel513 — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**